deploy/helm/sumologic/values.yaml

```diff
@@ -1588,7 +1591,7 @@ otelcol:
     ## Memory Ballast size should be max 1/3 to 1/2 of memory.
     memBallastSizeMib: "683"
     image:
-      name: "sumologic/opentelemetry-collector"
+      name: "ghcr.io/sumologic/opentelemetry-collector"
```

Comment:
Is this automatically published somehow?

Reply:
Nope, I moved the images by hand for now. We'd need to automate that for future use.

Reply:
Task for automating this: https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/1175